action.yml

```diff
@@ -53,7 +53,7 @@ runs:
         export PROPPROOF_PATH=$HOME/propproof
         git clone https://github.com/model-checking/kani \
```

Comment:
do you mind changing this to depth 0 please?

Reply:
actually, we should probably just be using the checkout action

Reply:
Changed to checkout action.